requirements.txt

```diff
@@ -32,7 +32,7 @@ pycodestyle==2.0.0
 pyflakes==1.2.3
 Pygments==2.1.3
 pyproj==1.9.5.1
-pyramid==1.8.3
+pyramid==1.9.1
```

Comment:
Is this upgrade save?

Reply:
Use my PR https://github.com/geoadmin/mf-chsdi3/pull/3086 (Pyramid 1.10)